creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -1034,6 +1082,43 @@ class SnapbackSM {
       this.snapbackReconfigEnabled = this.nodeConfig.get('snapbackReconfigEnabled')
     }
   }
+
+  // Fetch clock values with max number of attempts
+  async fetchClockValues (secondary1, secondary2, wallet, maxClockFetchAttempts = 50) {
```

Comment:
will update comment

fetches clock values for input secondaries
will default to -1 if could not access clock value at the time
so response will look like:
```
{
    secondary1: -1 or some positive integer,
    secondary2: -1 or some positive integer
}
```

Reply:
will update this method after discussion